recipe/bld.bat

```diff
@@ -0,0 +1,13 @@
+python -m pip install --no-deps --ignore-installed .
+
+REM Get Python sys.prefix
+FOR /F "tokens=* USEBACKQ" %%F IN (`python -c "import sys; print(sys.prefix)"`) DO (
+    SET JUPYTERLAB_DIR=%%F\share\jupyter\delux
+)
+
+REM Extensions to install
+jupyter labextension install @jupyter-widgets/jupyterlab-manager --no-build
+jupyter labextension install bqplot --no-build
+jupyter lab build
+jupyter lab clean
```

Comment:
I don't quite understand why `jupyter lab clean` is appropriate here. Doesn't that delete the installed extensions in the appdir?

Reply:
Currently `jupyter lab clean` is only removing the `staging` directory (see https://github.com/jupyterlab/jupyterlab/blob/master/jupyterlab/commands.py#L264).
And removing it is a big improvement regarding archive size (it reduces the package size by more than 25Mo for me).

But there is some discussion there: https://github.com/jupyterlab/jupyterlab/issues/4930

So should we keep `jupyter lab clean` or explicitly removing the `staging` directory.

Reply:
Thanks for clarifying, I was going based on the documentation.  Leaving it as `clean` makes sense, but we should keep an eye on jupyterlab/jupyterlab#4930 in case they fix the discrepancy in the code instead of the docs 🙂